io.openems.edge.controller.ess.predictivedelaycharge/src/io/openems/edge/controller/ess/predictivedelaycharge/ac/Config.java

```diff
@@ -0,0 +1,30 @@
+package io.openems.edge.controller.ess.predictivedelaycharge.ac;
+
+import org.osgi.service.metatype.annotations.AttributeDefinition;
+import org.osgi.service.metatype.annotations.ObjectClassDefinition;
+
+@ObjectClassDefinition( //
+		name = "Controller Predictive Delay Charge AC", //
+		description = "controller which calculates the power limitations every second to optimize the charging from PV")
+@interface Config {
+
+	@AttributeDefinition(name = "Component-ID", description = "Unique ID of this Component")
+	String id() default "ctrlPredictiveDelayChargeAC0";
+
+	@AttributeDefinition(name = "Alias", description = "Human-readable name of this Component; defaults to Component-ID")
+	String alias() default "";
+
+	@AttributeDefinition(name = "Is enabled?", description = "Is this Component enabled?")
+	boolean enabled() default true;
+
+	@AttributeDefinition(name = "Ess-ID", description = "ID of Ess device.")
+	String ess_id() default "ess0";
+
+	@AttributeDefinition(name = "Grid-Meter-Id", description = "ID of the Grid-Meter.")
+	String meter_id() default "meter0";
+
+	@AttributeDefinition(name = "buffer hour", description = "number of hours before the final hour.")
+	int Buffer_hours() default 2;
```

Comment:
Description is not sufficient to understand in my opinion. Also names of variables should be in camel-case starting with lower case